### PR TITLE
Implement zero-rating mode except for reCaptcha and ToS page. #7

### PR DESCRIPTION
**Pull request by gimite**
I disabled all services I can think of which loads external resource, except for reCaptcha and ToS. But I'm not really sure these are all. I will check it with production after submitting this.

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -395,6 +395,19 @@ def setup_env(request):
         # support UTF-8 but don't render UTF-8 symbols such as u'\xab'.
         env.back_chevron = u'<<'
 
+    env.enable_maps = (
+        env.enable_javascript
+        and not env.config.zero_rating_mode
+        and env.maps_api_key)
+    env.enable_analytics = (
```
Comment (skywhale) [inline app/main.py]: I don't have full context here, but external api calls are totally prohibited with zero-rating mode, or is this to save bandwidth?

File: app/resources/admin.html.template [hunk 2]
```diff
@@ -899,6 +899,36 @@
...
 
   <fieldset>
+    <legend>Zero-rating</legend>
+
+    <div class="config">
+      <p>Sometimes mobile carriers "zero-rate" our site so that users can see our site without normal data access fee. It is often applied per domain name. But when users access Person Finder, it also loads resource from other domains e.g., Google Maps, Google Analytics, Google Translate by default.</p>
+
+      <p>If you enable zero-rating mode, it disables all those features which loads resource from external domains, so that the users from the carriers can access Person Finder completely for free.</p>
+
+      <p>CAVEAT: Even if you enable zero-rating mode, the photos attached to person records or notes may be loaded from external domains, if the author has specified a URL of the photo instead of uploading it.</p>
```
Comment (skywhale) [inline app/resources/admin.html.template]: Do you want to file a feature request for routing such images through personfinder domain?

File: app/main.py (hunk 1 above)
Comment (skywhale) [inline app/main.py]: Sorry, I just read your description below. My question answered. Do we currently use Analytics for understanding user behavior? I assume not, and it's fine to disable this for zero-rating users?

**Comment by skywhale** [comment]
LGTM. Thank you!

File: app/main.py (hunk 1 above)
Comment (gimite) [inline app/main.py]: We do use Analytics to analyze the usage. So there's a trade off between enabling zero-rating mode and giving up analysis via Analytics, which should be decided per crisis.

File: app/resources/admin.html.template (hunk 2 above)
Comment (gimite) [inline app/resources/admin.html.template]: Good point. Done: https://github.com/google/personfinder/issues/218

**Pull request merged**